lib/global-admin/addon/components/cru-cloud-credential/component.js

```diff
@@ -380,9 +382,11 @@ export default Component.extend(ViewNewEdit, {
   },
 
   setError(message = '') {
+    set(this, 'cloudErrorExists', true);
     const translation = this.intl.t('modalAddCloudKey.errors.validation', { status: message });
 
     set(this, 'validatingKeys', false);
+    set(this, 'cloudError', this.intl.t('modalAddCloudKey.errors.errorMessage'));
 
     this.parseAndCollectErrors(translation, true);
```

Comment:
looks like they didn't implement this function. We should fix this to set the error